capsules/object_speed/stream_state.py

```diff
@@ -0,0 +1,43 @@
+from collections import OrderedDict
+from typing import Union, Tuple
+from uuid import UUID
+
+from vcap import BaseStreamState
+
+
+class StreamState(BaseStreamState):
+    """Basically an LRU cache that stores the last-seen timestamp and
+    coordinate keyed to detection.track_id
+    """
+
+    def __init__(self):
+        self.cache = OrderedDict()
+        """How many detection's information it can keep in the cache"""
+
+    def get(self, key: UUID) -> Union[Tuple[float, Tuple[int]],
+                                      Tuple[None, None]]:
```

Comment:
Not directly related to this PR, but we might want to consider returning objects with attributes instead of nameless tuples

Reply:
This is a custom object built just for this capsule. It _is_ relevant to the PR.

Maybe I'll make it a dataclass or something.

Reply:
Okay, I pushed a change where it now returns and expects a dataclass